extensions/eda/plugins/event_source/insights.py

```diff
@@ -26,6 +26,57 @@
 
 from aiohttp import web
 
+DOCUMENTATION = r"""
+---
+short_description: Receive Red Hat Insights events.
+description:
+    - An ansible-rulebook event source module for receiving Red Hat Insights events.
+options:
+    host:
+        description:
+            - The hostname to listen to. Set to 0.0.0.0 to listen on all  interfaces.
```

Comment:
Are these supposed to be a list of descriptions?

Reply:
According to this example https://github.com/ansible/ansible-documentation/blob/devel/examples/DOCUMENTATION.yml#L9-L11 it can be list.
Also running `ansible-doc -M extensions/eda/plugins/event_source --json insights -vvv` did not complain about it.